boxsdk/client/client.py

```diff
@@ -539,6 +539,251 @@ def create_group(self, name):
+            retention_attributes['disposition_action'] = disposition_action
+
+        if can_owner_extend_retention is not None:
+            retention_attributes['can_owner_extend_retention'] = can_owner_extend_retention
+        if are_owners_notified is not None:
+            retention_attributes['are_owners_notified'] = are_owners_notified
+        if custom_notification_recipients is not None:
+            retention_attributes['custom_notification_recipients'] = custom_notification_recipients
+        box_response = self._session.post(url, data=json.dumps(retention_attributes))
+        response = box_response.json()
+        return self.translator.translate(response['type'])(session=self._session, object_id=response['id'], response_object=response)
+
+    def get_retention_policies(
+            self,
+            policy_name=None,
+            policy_type=None,
+            created_by_user_id=None,
+            limit=None,
+            marker=None,
+            fields=None,
+    ):
+        """
+        Get the entries in the retention policy using marker-based paging.
+
+        :param policy_name:
+            The name of the retention policy.
+        :type policy_name:
+            `unicode` or None
+        :param policy_type:
+            Set to either `finite` or `indefinite`
+        :type policy_type:
+            `unicode` or None
+        :param created_by_user_id:
+            A user id to filter the retention policies.
+        :type created_by_user_id:
+            `unicode` or None
+        :param limit:
+            The maximum number of entries to return per page. If not specified, then will use the server-side default.
+        :type limit:
+            `int` or None
+        :param marker:
+            The paging marker to start paging from
+        :type marker:
+            `unicode` or None
+        :param fields:
+            List of fields to request
+        :type fields:
+            `Iterable` of `unicode`
+        :returns:
+            An iterator of the entries in the retention policy
+        :rtype:
+            :class:`BoxObjectCollection`
+        """
+        additional_params = {}
+        if policy_name is not None:
+            additional_params['policy_name'] = policy_name
+        if policy_type is not None:
+            additional_params['policy_type'] = policy_type
+        if created_by_user_id is not None:
+            additional_params['created_by_user_id'] = created_by_user_id
+        return MarkerBasedObjectCollection(
+            session=self._session,
+            url=self._session.get_url('retention_policies'),
+            additional_params=additional_params,
+            limit=limit,
+            marker=marker,
+            fields=fields,
+            return_full_pages=False,
+        )
+
+    def get_file_version_retentions(
+            self,
+            file_id=None,
+            file_version_id=None,
+            policy_id=None,
```

Comment:
All of these params should take objects (e.g. `File` or `RetentionPolicy`) for consistency with the rest of the SDK